.editorconfig

```
@@ -1,6 +1,6 @@
 [*]
 indent_style = space
-indent_size = 4
-
-[.github/workflows/*.yml]
 indent_size = 2
+
+[*.py]
+indent_size = 4
```

We should also add a 4 space indent for the spec files since they are not formatted automatically and aren't changed in this patch:

```suggestion
indent_size = 4

[scripts/core/*]
indent_size = 4
```

---

That's a good idea.

You happy about https://github.com/cheshirekow/cmake_format for the cmake?

---

> You happy about https://github.com/cheshirekow/cmake_format for the cmake?

If we do switch to that I think it should be separate from the PR, its already very large.